### ISSUE TITLE
latex drawer support for cregbundle

### ISSUE THREAD

**Issue by 1ucian0** [Qiskit/qiskit-terra#4291]
In the spirit of https://github.com/Qiskit/qiskit-terra/issues/2797 it would be great if latex drawer add support for `cregbundle`.

Take the following example:
```python
from qiskit import *
qr = QuantumRegister(3, 'q')
cr = ClassicalRegister(3, 'c')
circuit = QuantumCircuit(qr, cr)
circuit.measure(qr, cr)
circuit.draw('latex')
```
<img width="427" alt="Screen Shot 2020-04-27 at 3 10 31 PM" src="https://user-images.githubusercontent.com/766693/80410889-43f93200-8899-11ea-9bed-558804fa26c6.png">

I suggest something like this:
```
circuit.draw('latex', cregbundle=True)
```
<img width="451" alt="Screen Shot 2020-04-27 at 3 11 26 PM" src="https://user-images.githubusercontent.com/766693/80410958-612e0080-8899-11ea-9cce-5005f5ef43b1.png">

The latex code for that figure is:
```latex
\begin{equation*}
    \Qcircuit @C=1.0em @R=1.0em @!R {
	 	\lstick{ {q}_{0} :  } & \qw & \meter & \qw & \qw & \qw & \qw\\
	 	\lstick{ {q}_{1} :  } & \qw & \qw & \meter & \qw & \qw & \qw\\
	 	\lstick{ {q}_{2} :  } & \qw & \qw & \qw & \meter & \qw & \qw\\
	 	\lstick{c:} & {/_{_3}} \cw & \dstick{0} \cw \cwx[-3] & \dstick{1} \cw \cwx[-2] & \dstick{2} \cw \cwx[-1] & \cw & \cw\\
	 }
\end{equation*}
```

**Comment by rao107** [comment]
Hello! I haven't contributed to any open-source software but I want to try my hand at this.

**Comment by 1ucian0** [comment]
Sure! Go ahead. Let me know if I can help!

**Comment by 1ucian0** [comment]
It seems there is a small bug:
```python
cr1 = ClassicalRegister(2, 'c1')
cr2 = ClassicalRegister(2, 'c2')
qr = QuantumRegister(2, 'q')
circuit = QuantumCircuit(qr, cr1, cr2)
circuit.measure(qr[0], cr1[0])
circuit.measure(qr[0], cr1[1])
circuit.measure(qr[0], cr2[0])
circuit.measure(qr[0], cr2[1])
```
with `circuit.draw('latex', cregbundle=False)`:
![image](https://user-images.githubusercontent.com/766693/82233088-74167c80-98fd-11ea-9e9a-5fbffb4b1dab.png)

with `circuit.draw('latex', cregbundle=True)`:
![image](https://user-images.githubusercontent.com/766693/82233149-8a243d00-98fd-11ea-8fd0-445d458fc652.png)

The expected result is to have two registers in the last image.

A side note (if it's not easy to fix, no worries. We can leave it for later). Would it be possible avoid moving the gates right? Consider this:
```python
circuit = QuantumCircuit(1, 1)
circuit.h(0)
circuit.measure(0, 0)
circuit.draw('latex', cregbundle=False)
```
![download_1](https://user-images.githubusercontent.com/766693/82232493-b55a5c80-98fc-11ea-9410-685fa8724cd4.png)
```python
circuit = QuantumCircuit(1, 1)
circuit.h(0)
circuit.measure(0, 0)
circuit.draw('latex', cregbundle=True)
```
![download_2](https://user-images.githubusercontent.com/766693/82232519-be4b2e00-98fc-11ea-87d9-8c4b518239dd.png)

**Comment by rao107** [comment]
Oh wow, I never realized that multiple classical registers could be in the same circuit! That's pretty interesting. I'm working on squashing that bug now.

Relating to the side note: I'll certainly try my best to fix the formatting and avoid moving the gates to the right but, even from a brief look, the problem looks more complicated than I can handle. It's not going to stop me from trying though!

Unrelated to the bugs brought up, I found another bug where the number on the bundle would bug out:
```
circuit = QuantumCircuit(1, 10)
circuit.measure(0,0)
circuit.draw('latex', cregbundle=True)
```
![image](https://user-images.githubusercontent.com/54015293/82312172-8931f080-997b-11ea-843b-a2c9009db5b1.png)
But I then I fixed it:
![image](https://user-images.githubusercontent.com/54015293/82312437-e5951000-997b-11ea-9710-5819b09313fe.png)
I just wanted to let you know that I know about this and I've already dealt with it in case you also find the same bug as well.
